ensime-expand-region.el

```diff
@@ -0,0 +1,61 @@
+;;; ensime-expand-region.el --- Ensime expansions for Expand Region  -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2015  Sebastian Wiesner
+
+;; Author: Sebastian Wiesner <swiesner@lunaryorn.com>
+
+;; This file is not part of GNU Emacs
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Add Ensime expansions to Expand Region.
```

Comment:
since this will be a standalone file, can you please add instructions of use here?